src/helpers/telegram/commands/backup.js

```diff
@@ -10,15 +10,15 @@ module.exports = (ctx) => {
 	const args = command.splitArgs
 
 	let target = { id: user.id.toString(), name: user.first_name }
-	if (!_.includes(controller.config.telegram.admins, user.id.toString()) && ctx.update.message.chat.type === 'group') {
+	if (!_.includes(controller.config.telegram.admins, user.id.toString()) && ctx.update.message.chat.type === 'group' || ctx.update.message.chat.type === 'supergroup') {
 		return ctx.telegram.sendMessage(user.id, 'Please run commands in Direct Messages').catch((O_o) => {
 			controller.log.error(O_o.message)
 		})
 	}
-	if (_.includes(controller.config.telegram.admins, user.id.toString()) && ctx.update.message.chat.type === 'group') target = { id: ctx.update.message.chat.id.toString(), name: ctx.update.message.chat.title }
+	if (_.includes(controller.config.telegram.admins, user.id.toString()) && ctx.update.message.chat.type === 'group' || ctx.update.message.chat.type === 'supergroup') target = { id: ctx.update.message.chat.id.toString(), name: ctx.update.message.chat.title }
```

Comment:
change all to `controller.config.telegram.admins.includes(user.id.toString()) && (ctx.update.message.chat.type === 'group' || ctx.update.message.chat.type === 'supergroup')` note the parenthesis

Reply:
Thats what i thought. fixed :)